src/collector.ts

```diff
@@ -140,10 +150,27 @@ class NaiveGomodParser {
 /* Process entries found in the go.mod file and collect all dependency
  * related information */
 class GomodDependencyCollector implements IDependencyCollector {
-    constructor(public classes: Array<string> = ["dependencies"]) {}
+    constructor(private manifestFile: string, public classes: Array<string> = ["dependencies"]) {
+        this.manifestFile = manifestFile
+    }
 
     async collect(contents: string): Promise<Array<IDependency>> {
-        let parser = new NaiveGomodParser(contents);
+        let promiseExec = new Promise<Set<string>>((resolve, reject) => {
+            const vscodeRootpath = this.manifestFile.replace("file://", "").replace("/go.mod", "")
+            exec(`go list -f '{{ join .Imports "\\n" }}' ./...`,
+                   { cwd: `${vscodeRootpath}`, maxBuffer: 1024 * 1200 }, (error, stdout, stderr) => {
+                if (error) {
+                    console.log("Fail to execute: [", `go list -f '{{ join .Imports "\\n" }}' ./...`, `] at "${vscodeRootpath}"`)
+                    reject("'go list' command failed with error :: " + stderr.toString())
```

Comment:
I see mix of ``, '', "". Can you make it consistent using ``?
```suggestion
                    reject(`'go list' command failed with error: ${stderr.toString()}`);
```